programs/cleos/main.cpp

```diff
@@ -358,30 +350,34 @@ chain::action generate_nonce_action() {
    return chain::action( {}, config::null_account_name, name("nonce"), fc::raw::pack(fc::time_point::now().time_since_epoch().count()));
 }
 
-auto abi_serializer_resolver_empty = [](const name& account) -> std::optional<abi_serializer> {
-   return std::optional<abi_serializer>();
-};
-
 //resolver for ABI serializer to decode actions in proposed transaction in multisig contract
 auto abi_serializer_resolver = [](const name& account) -> std::optional<abi_serializer> {
-   static unordered_map<account_name, std::optional<abi_serializer> > abi_cache;
-   auto it = abi_cache.find( account );
-   if ( it == abi_cache.end() ) {
+  static unordered_map<account_name, std::optional<abi_serializer> > abi_cache;
```

Comment:
this function got its indentation butchered from 3 spaces to 2 spaces, let's restore it